src/sentry/receivers/releases.py

```diff
@@ -113,9 +113,14 @@ def resolved_in_commit(instance, created, **kwargs):
 
                 if user_list:
                     acting_user = user_list[0]
-                    self_assign_issue = UserOption.objects.get_value(
-                        user=acting_user, key="self_assign_issue", default="0"
+                    # TODO(hybrid-cloud): rely on user options being returned in the user service get calls.
```

Comment:
Is this something we should be doing now? Otherwise we'll have to go back over this soon and remove the user_option service we just created.

Reply:
IMO it's fine to treat this as a second unit of work when we come back to implement the actual user service and fully understand the scope.